### PR TITLE
Signup business integration

### PR DESCRIPTION
**Pull request by carlthronson**

### CODE REVIEW

File: src/components/modals/signupModals/signupBusiness1.tsx [hunk 1]
```diff
@@ -121,6 +121,12 @@ export default function SignupModalBusiness1() {
             colorScheme="c1"
             aria="submit"
             onClick={handleSubmit((data) => {
+              // It looks like the only data we need to signup a business
+              // Is business (business name)
+              // email
```
Comment (carlthronson) [inline src/components/modals/signupModals/signupBusiness1.tsx:126]: Can I assume the email will be unique in the database?

File: src/components/modals/signupModals/signupBusiness1.tsx [hunk 2]
```diff
@@ -121,6 +121,12 @@ export default function SignupModalBusiness1() {
             colorScheme="c1"
             aria="submit"
             onClick={handleSubmit((data) => {
+              // It looks like the only data we need to signup a business
+              // Is business (business name)
```
Comment (carlthronson) [inline src/components/modals/signupModals/signupBusiness1.tsx:125]: Can I assume the business (name) will be unique in the database?

**Comment by carlthronson** [comment]
It looks like the data for signing up a business is betaTester, earlySignup, business and email.  What should happen if email and/or business are already in the database?

**Comment by jennysukut** [comment]
@carlthronson If the business is already in the database {say, they already gave a donation so their businessName + contactName + donatedAmount is in our system}, is there a way we would be able to find their data and modify it with the "betaTester" and "earlySignup" options? 

Actually, I think we should pre-assign all the businesses who donate with "earlySignup" so they have access to entry earlier. The "betaTester" would be the only field that would need to be updated for a pre-existing business.

**Comment by carlthronson** [comment]
> @carlthronson If the business is already in the database {say, they already gave a donation so their businessName + contactName + donatedAmount is in our system}, is there a way we would be able to find their data and modify it with the "betaTester" and "earlySignup" options?
> 
> Actually, I think we should pre-assign all the businesses who donate with "earlySignup" so they have access to entry earlier. The "betaTester" would be the only field that would need to be updated for a pre-existing business.

Yes.  We can find their data.  And if the business is already in our database, we will update the fields that changed.

There can be some interesting use cases if we explore this further.  But I think what we are heading towards now is very straight forward and makes the most sense for the MVP

**Pull request closed without merging**